docs/application/vstools/tools/emulator-manager.md

```diff
@@ -1,152 +1,163 @@
 
-    **Figure: Launch the Emulator Manager in the Tools menu**
+>**Note**
+>
+>Emulator Manager for Visual Studio has the same requirement as Emulator Manager in Tizen Studio. For more information on hardware and software requirements, see [Emulator Requirements](../../tizen-studio/setup/prerequisites.md#emulator).
+
+After installing Emulator Manager from Visual Studio, you can launch it in one of the following ways: 
 
-    ![Launch the Emulator Manager in the Tools menu](media/em-vstoolbar1.png)
+- In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
 
--   On the Visual Studio toolbar, click **Launch Tizen Emulator**.
+     ![Launch the Emulator Manager in the Tools menu](media/em-vstoolbar1.png)
 
-    **Figure: Launch the Emulator Manager in Visual Studio**
+- In the Visual Studio toolbar, click **Launch Tizen Emulator**.
 
     ![Launch from the Visual Studio toolbar](media/em-vstoolbar2.png)
 
 
 <a name="create"></a>
-## Creating Emulator Instances
-
-The Emulator Manager can help you to select the recommended platform and template. When you need another device environment, you can edit an existing emulator instance, or create a new one with a more suitable platform and template. You can also create new platforms and templates to suit your needs.
+## Create Emulator Instance
 
-**Figure: Emulator Manager**
+While creating an emulator instance, you must have a [platform](#create-platforms) and a [template](#create-templates) with device definitions.
+Emulator Manager helps you to select the recommended platform and template to emulate your application. If you need some other device environment, you can create a new one or you can edit an existing emulator instance. You can also create new platforms and templates according to your requirement. 
 
-![Emulator Manager](media/em-vs1.png)
+![Emulator Manager](media/create_em_vs1.gif)
 
-To create a new emulator instance:
+To create an emulator instance, follow these steps:
 
-1.  In the Emulator Manager, click **Create**.
-2.  Select the platform (system image), and click **Next**.
-3.  Select the template (device definition), and click **Next**.
-4.  Modify the properties as needed, and click **Finish**.
+1.  In Emulator Manager, click **Create**. 
+2.  In the **Emulator Configuration** window that appears, select a system image, and click **Next**.
+3.	Select a device definition and click **Next**. 
+4.	Define the emulator properties and click **Finish**.
 
-The emulator instance appears in the Emulator Manager.
+The emulator instance appears in Emulator Manager.
 
-To view the emulator instance details, right-click the instance and select **Detail**.
+If you want to view the created emulator instance details, right-click the instance and click **Detail**.
```

Comment:
We are using "select" when we select one among options.
Please use the standard term. We didn't describe this rule because writers can realize if they read a few Tizen documents.

Reply:
Done